pkg/minikube/out/out.go

```diff
@@ -113,17 +113,32 @@ func Styled(st style.Enum, format string, a ...V) {
 	}
 }
 
-// Boxed writes a stylized and templated message in a box to stdout
-func Boxed(format string, a ...V) {
+func boxedCommon(format string, a ...V) (string, box.Box) {
 	str := Sprintf(style.None, format, a...)
 	str = strings.TrimSpace(str)
 	box := box.New(box.Config{Py: 1, Px: 4, Type: "Round"})
 	if useColor {
 		box.Config.Color = "Red"
 	}
+	return str, box
+}
+
+// Boxed writes a stylized and templated message in a box to stdout
+func Boxed(format string, a ...V) {
+	str, box := boxedCommon(format, a...)
 	box.Println("", str)
```

Comment:
how come the normal box does not need range lines ?

Reply:
Fixed